Singer.API/ClientApp/src/app/modules/core/core.module.ts

```diff
@@ -42,7 +42,8 @@ export const MY_FORMATS = {
       AgegroupToColorPipePipe,
       UserCardComponent,
       SingleRegistrationComponent,
-      DailybasisRegistrationsComponent
+      DailybasisRegistrationsComponent,
+      DetailedUserCardComponent,
```

Comment:
This component should belong to the `SharedModule` instead of the `CoreModule`.
Same is true for the `UserCardComponent`, but when that was created we didn't have a `SharedModule` yet

Reply:
The components have been moved to the Shared Module.